### DISCUSSION TITLE
[ FF7 ] Game crashes when selecting NEW GAME (FFNx gives an error)

### DISCUSSION THREAD

**Discussion by DrakeRyzer** [julianxhokaxhiu/FFNx#186] [General]
Wanted to play FF7 again after a few months.  After updating 7th Heaven and the various mods I had installed, along with adding a few new ones.  Whenever I select NEW GAME on the start menu it gives me this error.  I have tried disabling mods one at a time and it still crashes, with any combination of mods, as well as reinstalling both FF7 and 7H fully.  If I disable every mod it works however.

![Final Fantasy VII 9_26_2020 12_52_45 PM](https://user-images.githubusercontent.com/71940233/94346251-4ce16180-fff9-11ea-917c-b2f672b38ac6.png)

Requested files
[crash.ff7](https://1drv.ms/u/s!Ai5MbW2FHkLOoohKA_bqEr9GTMRlfg?e=ciOUgs)
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/5287058/FFNx.log)

List of mods I have enabled
![7th Heaven v2 2 3 522 - Mod Manager for Final Fantasy 7  Default  9_26_2020 1_09_19 PM](https://user-images.githubusercontent.com/71940233/94346294-93cf5700-fff9-11ea-8048-8c7751935170.png)

**Comment by julianxhokaxhiu** [comment]
Hi,

Thanks for the bug report but actually this has nothing to do with FFNx itself. Please approach the 7th author on Qhimm forums or Discord. My personal opinion is that you may have the 60fps battle mod enabled. Try disabling it :)

Thanks for using FFNx.

Best regards,
Julian

**Comment by DrakeRyzer** [comment]
Disabling 60fps battles didn't change anything.  Thanks for the help though, I will try the 7H forums.
